### ISSUE TITLE
[OAKTrust] Community search page

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#216]
**URL:** https://oaktrust-pre.library.tamu.edu/communities/ed9a1370-076a-4cc0-bf87-25ae04053a36/search
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.3.1 Info and Relationships (A)
**Issues:**  Two H1 designations
**Notes:** Both "Research Projects" and "Search Results" are labeled H1

**Comment by devangm** [comment]
During the meeting, we notified the customer of potential limitations in implementing a single `<h1>` tag on the search results page due to the modular design of DSpace.

Example:
On the [Home page](https://oaktrust-pre.library.tamu.edu/home),  both “Colleges and Schools” and “Search Results” are rendered as `<h1>` tags. If we update “Search Results” to `<h2>`, the same component on the [Search Results page](https://oaktrust-pre.library.tamu.edu/search?spc.page=1) would also display as `<h2>`, resulting in that page having no `<h1>` at all.

William suggested implementing logic to dynamically render the “Search Results” component as `<h1>` or `<h2>` depending on the page context or content being displayed. Bonnie approved this solution.

**Comment by tamu-sad-iii** [comment]
Add hidden h1 on search page and to use h2 for search result component.
